content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/inventory_valuation_config.rst

```diff
@@ -117,26 +134,52 @@ value, and any product moves from then on affect the average cost, and the cost
 change. If the value in the :guilabel:`Cost` field on a product form is changed manually, Odoo will
 generate a corresponding record in the *Inventory Valuation* report.
```

Comment:
```suggestion
change. If the value in the :guilabel:`Cost` field on a product form is changed manually, Odoo
generates a corresponding record in the *Inventory Valuation* report.
```

Reply:
Manually applied this change on my end (: